### PR TITLE
[StepRepresentationLicence] Modify disposition

### PR DESCRIPTION
**Pull request by Tutul-**
+ Add missing properties for POL

It's ok now ? :)
capture d ecran de 2015-07-08 00 41 25

### CODE REVIEW

**Comment by qparis** [comment]
No, please have a look at TextField. It is not presented this way

**Comment by Tutul-** [comment]
capture d ecran de 2015-07-08 01 01 08

File: src/main/java/com/playonlinux/ui/impl/javafx/setupwindow/StepRepresentationLicence.java [hunk 1]
```diff
@@ -73,7 +65,7 @@ protected void drawStepContent() {
     @Override
     protected void setStepEvents() {
```
Comment (qparis) [inline src/main/java/com/playonlinux/ui/impl/javafx/setupwindow/StepRepresentationLicence.java]: You do not need to overwrite this method, do you?

**Comment by Tutul-** [comment]
It's better to put it for all methode from interface or other parent

**Comment by qparis** [comment]
No it is not. This is the whole point of inheritance: avoid duplicate code

**Comment by Tutul-** [comment]
I can remove it but why keep it on other StepRepresentation*

**Comment by qparis** [comment]
Because it needs to be defined at least once

**Pull request merged**